packaging/request-a-package/en.md

```diff
@@ -14,10 +15,29 @@ Please provide as much of the following information as possible:
 - Homepage
 - Why should this be included in the repository? If we already offer similar software in our repository **or third party**, please provide information on what your proposed software does that our existing offerings don't (*differentiators*).
 - Is it Open Source (yes/no)
- - If yes: Set the Tags to have "Package Requests"
- - If no: Set the Tags to have "Third Party Repository"
+  - If yes: Set the Tags to have "Package Requests"
+  - If no: Set the Tags to have "Third Party Repository"
 - How many users do you anticipate will use this software?
 - Link to source tarball/zip file
- - master.zip files **are not permitted**. We require versioned tarballs, for example: "1.2.3.tar.gz".
+  - master.zip files **are not permitted**. We require versioned tarballs, for example: "1.2.3.tar.gz".
+
+Please put this information into a Bug Report (create via "Create Task") on our Developer Portal via the Maniphest section on https://dev.solus-project.com/
+
+
+
+# Requesting A Package Update 
+
+Packages are normaly regularly updated through the package Maintainers, however if we are missing a package update you can let us know using our Task Tracker.
+
+**Please look to see if a update request has been filed for the software or library you require**. If there isn't an existing request, you can use the template below for requesting to update a package.
+
+Please provide the following information:
+
+- Title (Update $packagename to $version)
+- Short explenation text
```

Comment:
Should be:

- Description: Explanation as to the value-add of updating this package. Updates solely for a new version number, platform-specific updates (such as to macOS or Windows) which aren't relevant, or solely translation updates, are generally not accepted or not prioritized.